qa/suites/fs/upgrade/upgraded_client/from_pacific/tasks/2-workload/cfuse_dbench.yaml

```diff
@@ -0,0 +1,6 @@
+tasks:
+- ceph-fuse: [client.0]
```

Comment:
In `1-client-upgrade.yaml` has already include this:

```
  1 tasks:
  2 - install.upgrade:                                                                                                                                 
  3     client.0:
  4 - print: "**** done install.upgrade on client.0"
  5 - ceph-fuse:
  6     client.0:
  7 - print: "**** done remount client"
```
So here you are mounting the `ceph-fuse: [client.0]` twice.

Reply:
Yep, removed. Thanks!